src/main/java/org/takes/facets/hamcrest/AbstractHmHeader.java

```diff
@@ -44,6 +44,7 @@
  * @author I. Sokolov (happy.neko@gmail.com)
  * @author Andrey Eliseev (aeg.exper0@gmail.com)
  * @version $Id$
+ * @since 0.1
```

Comment:
@xupyprmv I cannot find this file in version `0.1`: https://github.com/yegor256/takes/find/0.1